src/util/fetch.js

```diff
@@ -1,15 +1,15 @@
 export const fetchJson = (url, options = {}) => {
-    const requestHeaders = new Headers({
+    const requestHeaders = options.headers || new Headers({
```

Comment:
It would be nice to throw an error or at least a warning if `option.headers` is not a `Headers` instance.
What's the comportment of fetch if it isn't Headers?